### ISSUE TITLE
Improving accessibility for the visually impaired

### ISSUE THREAD

**Issue by AlisonRumsey** [gctools-outilsgc/gcconnex#807]
The following are a few issues that could improve accessibility for the visually impaired: 
1. JAWS (screen reader) is unable to select the text box when adding a file. 
2. JAWS does not pick up on the Welcome tutorial that pops up upon initial sign in. 
3. JAWS does not always pick up on the "edit" field. ie you must click instead of using regular JAWS functions (similar to using tab)
4. Ctrl+v works in “Files” widget, but right click paste does not. Both work in “Blogs” widget. 
5. When using JAWS, and clicking on the "Messages" or "Blogs" tab, the page does not load.

#BreakingBarriers

**Comment by ToferC** [comment]
@Troy-Lawson - would be great to get these items into the next code sprint.
